src/v2/pages/channel/components/ChannelContentsWithData/index.tsx

```diff
@@ -49,25 +38,8 @@ export const ChannelContentsWithData: React.FC<Props> = WithIsSpiderRequesting<
         const { channel: clientChannel } = data
         const channel = { ...serverChannel, ...clientChannel }
 
-        const shouldSubscribeToPusher =
-          !isSpiderRequesting && channel.can.add_to
-
-        if (shouldSubscribeToPusher) {
```

Comment:
Perviously we were calling `setupPusherChannel` every time this component gets rendered, which creates new `pusherChannel ` and `socket` values that don't have reference equality across renders. This means that we were unnecessarily rendering `ChannelContents` a ton more than needed (ChannelContents is a pure component)